src/pages/App/App.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
So all the pages have to be in the page folder, even the App pages?

Reply:
Yea. you are right!